File: Informatyka sem 2 2016/Labolatorium 10 Zadanie 3-4/Zadanie3/src/zadanie3/Rectangle.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package zadanie3;

/**
 *
 * @author Lenovo
 */
public class Rectangle extends Shape
{
    private double width;
    private double height;
    public Rectangle(String n,double w,double h)
    {
        super(n);
        width=w;
        height=h;
    }
    public double area() 
    {
        return width*height;
    }
}

File: Informatyka sem 2 2016/Labolatorium 3 Zadanie 1/Labolatorium 3 Zadanie 1/Rectangle.cpp
#include "Rectangle.h"
#include<iostream>
using namespace std;
Rectangle::Rectangle()
{
	cout << "Konstruktor klasy Rectangle" << endl;
}
Rectangle::Rectangle(string n, double w, double h) : height(h), width(w), Shape()
{
	cout << "Konstruktor klasy Rectangle" << endl;
}
double Rectangle::SetWidth()
{
	return width;
}
double Rectangle::SetHeight()
{
	return height;
}
Rectangle::~Rectangle()
{
	cout << "Destruktor klasy Shape" << endl;
}
double Rectangle::Area()
{
	return height*width;
}
